src/molecules/BottomSheet/BottomSheetSectionList.native.js

```diff
@@ -0,0 +1,10 @@
+import { SectionList } from 'react-native';
+
+const BottomSheetSectionList = () => null;
+
+BottomSheetSectionList.displayName = 'BottomSheetSectionList';
+
+/**Not sure if this works :( */
+BottomSheetSectionList.propTypes = SectionList;
```

Comment:
This throws waring. Instead, we can define defaultProps?
`BottomSheetSectionList.defaultProps = SectionList;`